### DISCUSSION TITLE
配置文件的outbounds下可以配置多个vmess protocol吗？

### DISCUSSION THREAD

**Discussion by aoterbear** [v2fly/v2ray-core#742] [Q&A]
我的需求是netflix走一个节点，hbo走另一个节点，想到的办法是在outbounds下配置多个vemss protocol并配置不同的tag，这样在routing下就可以根据域名选择不同的tag以达到分流的目的，不过貌似这么配置后不生效。请问这个想法是否可行呢？

**Comment by kslr** [comment]
可以
